onnxruntime/core/providers/js/js_execution_provider.cc

```diff
@@ -92,6 +92,7 @@ class ONNX_OPERATOR_KERNEL_CLASS_NAME(kJsExecutionProvider, kOnnxDomain, 13, Erf
 class ONNX_OPERATOR_VERSIONED_KERNEL_CLASS_NAME(kJsExecutionProvider, kOnnxDomain, 6, 12, Sigmoid);
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kJsExecutionProvider, kOnnxDomain, 13, Sigmoid);
 
+class ONNX_OPERATOR_KERNEL_CLASS_NAME(kJsExecutionProvider, kOnnxDomain, 7, Not);
```

Comment:
Not is available since opset version1 unlike Sin, Cos, etc.

Reply:
It looks like the `npm test -- op not.jsonc -b=webgpu` actually runs CPU kernel because it's not correctly registered.